minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -994,7 +993,7 @@ public synchronized void stop() throws IOException, InterruptedException {
     control.stop(ServerType.SCAN_SERVER, null);
 
     // The method calls above kill the server
-    // Clean up the locks in ZooKeeper fo that if the cluster
+    // Clean up the locks in ZooKeeper for that if the cluster
```

Comment:
I'm not sure this sentence makes any more sense after this fix. "for that if the cluster is restarted" is a very confusing statement, and I'm not sure what it's trying to say.

Reply:
It looks like it should have been 'so' instead of 'for'.  Fixed in a481f9b.